PyPortal_Azure_Plant_Monitor/code.py

```diff
@@ -31,18 +38,36 @@
 esp32_reset = DigitalInOut(board.ESP_RESET)
 spi = busio.SPI(board.SCK, board.MOSI, board.MISO)
 esp = adafruit_esp32spi.ESP_SPIcontrol(spi, esp32_cs, esp32_ready, esp32_reset)
+
+# Set up the WiFi manager with a status light to show the WiFi connection status
 status_light = neopixel.NeoPixel(board.NEOPIXEL, 1, brightness=0.2)
 wifi = adafruit_esp32spi_wifimanager.ESPSPI_WiFiManager(esp, secrets, status_light)
+wifi.connect()
```

Comment:
Could you add a `print` before this to show the user that the WiFi is connecting, and a `print` statement after displaying it's connected? It's often hard to determine the state of the code when the display is off.